facade/src/main/java/org/jboss/pnc/facade/providers/BuildConfigurationProviderImpl.java

```diff
@@ -308,6 +335,7 @@ public void addDependency(String configId, String dependencyId) {
 
         logger.debug("Didn't throw any validation errors");
         buildConfig.addDependency(dependency);
+        buildConfig.setLastModificationUser(user);
```

Comment:
Dependencies are not audited. Should we update the user even when no audited data got changed?

Reply:
I would have done that by default, if anything got changed in the BuildConfiguration, I would have updated the lastModificationUser, even if the changes were not audited. But that would indeed cause some confusion now, as users would see a lastModificationUser changed but possibly no corresponding revision in the list...so yep, better not updating it at the moment. Thanks!